sdks/python/apache_beam/internal/cloudpickle/__init__.py

```diff
@@ -0,0 +1,18 @@
+from . import cloudpickle
+from .cloudpickle import *  # noqa
```

Comment:
Nit: to be explicit I think it might be good to put this in `.../apache_beam/internal/vendor/...` instead of `.../apache_beam/internal/...`